x/validator/types/reward.go

```diff
@@ -72,6 +79,16 @@ func (r *Reward) AddSig(sig []byte, expectedSigner string) error {
 	return nil
 }
 
+// Add more reward
+func (r *Reward) Add(rewardType RewardType, amount sdk.Int) {
+	switch rewardType {
+	case ServiceReward:
+		r.ServiceReward = r.ServiceReward.Add(amount)
+	case MiningReward:
+		r.MiningReward = r.MiningReward.Add(amount)
+	}
+}
```

Comment:
where is this new function used? It seems a bit repetitive with the keeper.AddReward()

Reply:
It is not used. I forgot to remove it.